public/app/features/explore/LogsSamplePanel.tsx

```diff
@@ -31,6 +44,32 @@ export function LogsSamplePanel(props: Props) {
     });
   };
 
+  const OpenInSplitViewButton = () => {
+    if (!hasSupplementaryQuerySupport(datasourceInstance, SupplementaryQueryType.LogsSample)) {
+      return null;
+    }
+
+    const logSampleQueries = queries
+      .map((query) => datasourceInstance.getSupplementaryQuery(SupplementaryQueryType.LogsSample, query))
+      .filter((query): query is DataQuery => !!query);
+
+    if (!logSampleQueries.length) {
+      return null;
+    }
+
+    return (
+      <Button
+        size="sm"
+        className={styles.logSamplesButton}
+        // TODO: support multiple queries
```

Comment:
I was trying to understand this comment. Is this something we would need to do? Or are we just limited because you can only split once to a new query?

Reply:
We are limited now, but in the future we'd like to change API to split open, we should update this as well. I am going to add reference to the issue.